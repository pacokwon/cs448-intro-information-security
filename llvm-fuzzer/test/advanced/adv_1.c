<<<<<<< HEAD
int main()
{
    int x = 0;
    int t = 10;

=======
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char input[65536];
    int x = 0;
    int t = 10;

    fgets(input, sizeof(input), stdin);
    x = atoi(input);

>>>>>>> fuzzer/master
    if(x >= 10)
    {
        int y = 100 / x;
        int w = x / 0;
        t = 10 - x;
    }
    else if(x <= -10)
    {
        int y = 100 / x;

        t = 10 + x - x + x;
    }
    else if(x >= 0)
    {
        int y = 100 / x;
        t = t * x;
    }
    else
    {
        int y = 100 / x;
        t = t / y;
    }

    t = x / t;
}
