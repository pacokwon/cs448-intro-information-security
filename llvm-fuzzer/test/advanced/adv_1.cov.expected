6,10
6,8
12,15
12,13
18,15
18,13
